Dockerfile

```diff
@@ -7,8 +7,8 @@ ARG OSNICK=buster
 ARG ARCH=x64
 
 #----------------------------------------------------------------------------------------------
-# FROM redis:latest AS builder
-FROM redisfab/redis-${ARCH}-${OSNICK}:5.0.5 AS builder
+FROM redis:6.0-rc3 AS builder
```

Comment:
@ashtul @rafie won't this break ARM build?

Reply:
Should be `FROM redisfab/redis:6.0-rc3-${ARCH}-bionic AS builder`.

Reply:
Isn't the arm builder in file `Dockerfile.arm`?
For now, left as is since there is no `-xbuild` version.

Reply:
Well, it's a little bit funny. The Dockerfile.arm will build you an arm version when you're running it on an intel hardware, which is kind of cool. But what if you're running on arm hardware and want to build natively? Then you use Dockerfile, and the ARCH arg tells it which redis image it should get. Really, such a pity nobody uses it. :-)